main.py

```diff
@@ -1,10 +1,51 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
+with open("players.json", "r") as sourse:
+    data = json.load(sourse)
+
 
 def main() -> None:
-    pass
+    for nickname, info in data.items():
+        name_race = info["race"]["name"]
+        race_description = info["race"]["description"]
+
+        if not Race.objects.filter(name=name_race).exists():
+            race = Race.objects.create(
+                name=name_race,
+                description=race_description
+            )
```

Comment:
if race already exists, how do you get it?
For this, you can use the `get_or_create()` method. The same in other similar cases